AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -55,9 +55,10 @@ def reduce_compscan_inf(h5,rfi_static_flags=None,chunks=16,return_raw=False,use_
     calibrated = False # placeholder for calibration
     h5.select(compscans=compscan_index)
     h5.select(reset='B') # Resets only pol,corrprods,ants
-    active_ants = list(set(h5.antlist) & set(find_active_ants(h5, 0.85))) # Only those specified AND active during this compscan
+    active_ants = list(set([a.name for a in h5.ants]) & set(find_active_ants(h5, 0.85))) # Only those specified AND active during this compscan
```

Comment:
This looks like it can be simplified, since `find_active_ants` always starts from the selected `h5.ants` anyway. Then there is no need to AND it with that list.

```suggestion
    active_ants = find_active_ants(h5, 0.85)    # Only those specified AND active during this compscan
```

Reply:
you are forgetting that the function can be invoked with reduce_compscan_inf(h5), and h5 may have had "select()" applied to select only a subset of antennas.

Reply:
oops, that was my original thinking, but i see that the line preceding that actually  resets the antenna selection filter.
so you are right, your proposal is sound.

Reply:
`find_active_ants` work on the current `h5` object, which includes any selections. The `h5.ants` here and the `h5.ants` inside the function is the exact same list, so I can't see that this code will ever be needed.

Reply:
I see - we've removed the piggyback, which makes this use less obvious. If still needed, I would pass the list of included antennas explicitly to this function. Or reconstruct it from `h5.ants` instead of making your own `h5.antlist`.